src/test/java/io/kontur/disasterninja/dto/DtoTest.java

```diff
@@ -68,7 +68,10 @@ public void serializeDeserializeSummaryFromGeometryTest() {
         Assertions.assertEquals(layer.getGroup(), response.get(0).group());
         Assertions.assertEquals(layer.isBoundaryRequiredForRetrieval(),
                 response.get(0).boundaryRequiredForRetrieval());
-        Assertions.assertEquals(layer.getCopyrights(), response.get(0).copyrights());
+        Assertions.assertEquals(
+                layer.getCopyrights() != null ? layer.getCopyrights() : List.of(),
+                response.get(0).copyrights() != null ? response.get(0).copyrights() : List.of());
```

Comment:
Is this really necessary? Why replace `null`s with empty lists?

Reply:
I chose to replace `null` values with empty lists to ensure consistency and avoid potential `NullPointerException`s in tests. 

If there are any specific concerns or scenarios where `null` should be preserved, please let me know.

Reply:
I think if you replace `null`s with empty lists _in assertions_ you're testing the wrong thing. If a request returns a `null` it better should correspond to a `null` in the data, otherwise, it's a bug, and the test should catch it.

@palina-krukovich what do you think?

Reply:
reverted back old approach